design-library/src/components/BccHeader/BccHeader.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+type Props = {
+  overline?: string;
+  title?: string;
```

Comment:
I feel at least the title should be mandatory? Otherwise it won't render anything right 😅